igor-web/src/main/java/com/netflix/spinnaker/igor/gcb/CloudBuildFactory.java

```diff
@@ -24,12 +24,14 @@
 import com.google.api.services.cloudbuild.v1.CloudBuild;
 import lombok.RequiredArgsConstructor;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.stereotype.Component;
 
 /**
  * Factory for calling CloudBuild client library code to create CloudBuild objects
  */
 @Component
+@ConditionalOnProperty("gcb.enabled")
```

Comment:
Should `gcb` be the top level config family/namespace?

Reply:
That is how it is for the other CI tools in igor, though maybe it makes sense to start namespacing these a bit more?

Reply:
I was just curious since for pubsub we have a top-level grouping `pubsub`, maybe makes some sense to have one here as well?

Reply:
Chatted about this in person, we'll leave this top level to be consistent with other ci tools.